src/Core/Grid/Action/Row/Type/LinkRowAction.php

```diff
@@ -32,14 +32,14 @@
 /**
  * Class RowAction holds information related to row action for each row element in grid
  */
-final class SimpleRowAction extends AbstractRowAction
+final class LinkRowAction extends AbstractRowAction
 {
```

Comment:
imo you should keep SimpleRowAction available and introduce LinkRow as a new action type

Reply:
hmm, so `SimpleRowAction` would just add a button without url?